korebuild-lock.txt

```diff
@@ -0,0 +1,2 @@
+version:2.0.2-beta-15522
+commithash:f26cb086ebd8b5a187381bb9a431794f11d69025
```

Comment:
Going to need some tutorials on these changes to Korebuild when these updates are applied to all the `dev` branches :)

Reply:
Good to know. It'll be a few weeks before these changes make it to `dev`. I'll send instructions to the whole runtime team before these changes happen.